.azure/templates/phases/default-build.yml

```diff
@@ -94,7 +94,6 @@ phases:
   - ${{ if eq(parameters.artifacts.publish, 'true') }}:
     - task: PublishBuildArtifacts@1
       displayName: Upload artifacts
-      condition: eq(variables['system.pullrequest.isfork'], false)
```

Comment:
Suggest keeping this as:
```
condition: always()
```
This ensures artifacts are uploaded even if the tests failed. This is useful for debugging test failures.